src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -43,7 +43,7 @@ defineProperties(ArcGisMapServerItemViewModel.prototype, {
      */
     type : {
         get : function() {
-            return 'esri-mapServer';
+            return 'esri-rest';
```

Comment:
Not sure about CKAN, but Esri REST API docs lists a whole bunch of APIs.  We're only using the one called "Map Service":
http://resources.arcgis.com/en/help/rest/apiref/